Solutions/Zscaler Private Access (ZPA)/Analytic Rules/ZscalerSharedZPASession.yaml

```diff
@@ -0,0 +1,53 @@
+id: 40a98355-0e52-479f-8c91-4ab659cba878
+name: Zscaler - Shared ZPA session
+description: |
+  'Detects Shared ZPA session.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ZscalerPrivateAccess
+    dataTypes:
+      - ZPAEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1178
+  - T1133
+query: |
+  let open_sessions =
+  ZPAEvent
+  | where DvcAction == 'open'
+  | summarize timestampstart = min(TimeGenerated) by DstUserName, SrcIpAddr, NetworkSessionId
+  | sort by timestampstart asc;
+  let closed_sessions =
+  ZPAEvent
+  | where DvcAction == 'close'
+  | summarize timestampend = max(TimeGenerated) by DstUserName, SrcIpAddr, NetworkSessionId
+  | sort by timestampend asc;
+  open_sessions
+  | join (closed_sessions) on DstUserName, SrcIpAddr, NetworkSessionId
+  | serialize 

```

Comment:
serialize is not needed. Sort automatically mark a data set as serialized.

Reply:
fixed